docs/website/sidebars.js

```diff
@@ -136,6 +136,19 @@ const sidebars = {
         'reference/explainers/how-dlt-works',
         'general-usage/resource',
         'general-usage/source',
+        {
+          type: 'category',
+          label: 'Configuration and secrets',
+           link: {
+            type: 'doc',
+            id: 'general-usage/credentials/index',
+          },
+          items: [
+            'general-usage/credentials/setup',
+            'general-usage/credentials/advanced',
```

Comment:
Again, since we're renaming the sections anyway, maybe the `general-usage/configuration` path would be more consistent with the title of the page?

Reply:
hm, I don't think it that important, also there are far more info about credentials then configurations